plugin/ale.vim

```diff
@@ -33,17 +33,20 @@ if !s:has_features
 endif
 
 " Add the after directory to the runtimepath
+let s:sep = has('win32') ? '\' : '/'
 let s:should_add_after_dir = 1
 for s:path in split(&runtimepath, ',')
-    if s:path =~# '\/ale\/after$'
+    if s:path =~# s:sep . 'after$' &&
```

Comment:
This should check with `[\/]after$` instead since windows accepth both of the separators. For example, I often use slash to set runtimepath.

Reply:
ok, thank you. I'll fix it. And @thinca also gave me additional fix points.